template-processors/base/bin/resourceManager.sh

```diff
@@ -110,14 +115,9 @@ function createUpdateResources() {
     esac
```

Comment:
Would it make sense to add `None` to the `case` statement, so that's it's clear what's happening when this mode is set?

Reply:
```suggestion
    None)
        ;;
    esac
```